src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -605,6 +607,47 @@ private static String EscapeIDName(String inID) throws SQLServerException {
         return outID.toString();
     }
 
+    // Use LinkedHashMap to force retrieve elements in order they were inserted
+    private static final LinkedHashMap<Integer, String> getColumnsDWColumns = new LinkedHashMap<>();
+    static {
```

Comment:
I can see why you would want this block static, but I think a better option is to populate this map only if the connection is against Azure DW and it is empty.